uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -410,7 +411,12 @@ Arguments:
 sub PatientNameMatch {
     my $this         = shift;
     my ($dicom_file) = @_;
-    my $cmd          = "dcmdump $dicom_file | grep PatientName";
+
+    my $lookupCenterNameUsing = NeuroDB::DBI::getConfigSetting(
+        $this->{'dbhr'},'lookupCenterNameUsing'
+    );
+
+    my $cmd = "dcmdump +P $lookupCenterNameUsing $dicom_file";
```

Comment:
\<paranoid\>
`my $cmd = sprintf("dcmdump +p %s %s", quotemeta($lookupCenterNameUsing), quotemeta($dicom _file));`
\</paranoid\>

Reply:
Cool! Thanks!! Always good to be paranoid for these kind of things!!